editor/script_create_dialog.cpp

```diff
@@ -892,22 +892,41 @@ ScriptLanguage::ScriptTemplate ScriptCreateDialog::_parse_template(const ScriptL
 					script_template.description = line.substr(12, -1).strip_edges();
 				}
 				if (line.begins_with("space-indent")) {
-					String indent_value = line.substr(17, -1).strip_edges();
+					String indent_value = line.substr(13, -1).strip_edges();
```

Comment:
The length of the "space-indent" string is 12, but we take the substring from position 13 because the colon is at 12. The condition should be `line.begins_with("space-indent:")`.

Nitpick: The second argument of `substr()` is -1 by default and can be omitted.

Reply:
True, was written to match the other cases though, but will modify